src/wp-includes/rest-api/search/class-wp-rest-post-search-handler.php

```diff
@@ -58,12 +58,13 @@ public function search_items( WP_REST_Request $request ) {
 		}
 
 		$query_args = array(
-			'post_type'           => $post_types,
-			'post_status'         => 'publish',
-			'paged'               => (int) $request['page'],
-			'posts_per_page'      => (int) $request['per_page'],
-			'ignore_sticky_posts' => true,
-			'fields'              => 'ids',
+			'post_type'              => $post_types,
+			'post_status'            => 'publish',
+			'paged'                  => (int) $request['page'],
+			'posts_per_page'         => (int) $request['per_page'],
+			'ignore_sticky_posts'    => true,
+			'update_post_term_cache' => true,
+			'update_post_meta_cache' => true,
```

Comment:
```suggestion
			'update_post_term_cache' => true,
			'update_post_meta_cache' => true,
```

These are not needed, as these are the default values anyway.